public/app/features/dashboard/components/Inspector/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { saveAs } from 'file-saver';
+import { CollectorItem } from './types';
+import { safeStringifyValue } from '../../../../core/utils/explore';
+
+export interface Packager {
+  package: (items: CollectorItem[]) => string;
+}
+
+export function inspectPackager(): Packager {
+  return {
+    package: function (items: CollectorItem[]): string {
+      const markdowns: string[] = [];
+      const newline = '\n';
+
+      markdowns.push(`#### Shared data from Grafana (${new Date().toISOString()})`);
+      for (const item of items) {
+        let markdown = '';
+        markdown += `##### ${item.name}  `;
```

Comment:
FYI, OS and browser details were empty when I tried this

Reply:
Did you get build errors? Needed to add a new package for this functionality.

Reply:
Of course, you are right - only now I have realized there is a TS error in my console. nvm